api/auth/authproviders.py

```diff
@@ -66,7 +66,7 @@ def validate_code(self, code, **kwargs):
             'access_token': code,
             'uid': uid,
             'auth_type': self.auth_type,
-            'expires': datetime.datetime.utcnow() + datetime.timedelta(hours=1)
+            'expires': datetime.datetime.utcnow() + datetime.timedelta(days=14)
```

Comment:
I'm all for leaving the refresh logic alone - my question was specifically about why we can't respect the JWT expiry instead of setting our own duration?